gulpfile.js

```diff
@@ -485,6 +513,19 @@ gulp.task('regenerate:expected:csazurecomposite', function (cb) {
   }, cb);
 });
 
+
+gulp.task('regenerate:expected:go', function(cb){
+  regenExpected({
+    'outputBaseDir': 'src/generator/AutoRest.Go.Tests',
+    'inputBaseDir': 'src/generator/AutoRest.Go.Tests',
+    'mappings': goMappings,
+    'outputDir': 'src/tests/generated',
+    'codeGenerator': 'Go',
+    'flatteningThreshold': '1'
```

Comment:
>flatteningThreshold [](start = 5, length = 19)

Does Go generator really supports flattening threshold? or was it just copied from other gulp task?